Helper/Api/ApiService.php

```diff
@@ -45,40 +46,78 @@ public function __construct(LoggerInterface $logger, \Twig_Environment $twig, Ur
         $this->apiClients = $apiClients;
     }
 
-
-
     public function treatFormRequest(Request $request, string $apiName, string $validationTemplate = null)
     {
         $body = $request->request->all();
-        /** @var string $key */
-        /** @var UploadedFile $file */
-        foreach ($request->files as $key => $file) {
-            if ($file !== null) {
-                $response = $this->uploadFile($apiName, $file, $file->getClientOriginalName());
-                if (!$response['uploaded'] || !isset($response[EmsFields::CONTENT_FILE_HASH_FIELD_])) {
-                    throw new \Exception('File hash not found or file not uploaded');
-                }
-                $body[$key] = [
-                    EmsFields::CONTENT_FILE_HASH_FIELD => $response[EmsFields::CONTENT_FILE_HASH_FIELD_],
-                    EmsFields::CONTENT_FILE_HASH_FIELD_ => $response[EmsFields::CONTENT_FILE_HASH_FIELD_],
-                    EmsFields::CONTENT_FILE_NAME_FIELD => $file->getClientOriginalName(),
-                    EmsFields::CONTENT_FILE_NAME_FIELD_ => $file->getClientOriginalName(),
-                    EmsFields::CONTENT_FILE_SIZE_FIELD => $file->getSize(),
-                    EmsFields::CONTENT_FILE_SIZE_FIELD_ => $file->getSize(),
-                    EmsFields::CONTENT_MIME_TYPE_FIELD => $file->getMimeType(),
-                    EmsFields::CONTENT_MIME_TYPE_FIELD_ => $file->getMimeType(),
-                ];
-            }
-        }
-
+        $body = $this->treatFiles($body, $apiName, $request->files);
         if ($validationTemplate !== null) {
             return \json_decode($this->twig->render($validationTemplate, [
                 'document' => $body,
             ]), true);
         }
-
+        
         return $body;
     }
+    
+    /**
+     *
+     * @param array<string,mixed> $body
+     * @param string $apiName
+     * @param FileBag<array>|array<string,mixed> $files
+     * @return array<string,mixed>
+     */
+    private function treatFiles(array $body, string $apiName, $files): array
+    {
+        $body_field = $body;
```

Comment:
$bodyF
```suggestion
        $bodyField = $body;
```

Reply:
Did you test without $body_field = $body; ?

Just using $body directly. At first glance this seems like a possibility?

Reply:
Yes, $body_field must be deleted and use $body directly.